NARFO_BE/NARFO_BE/Controllers/ActivitiesController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using NARFO_BE.Models;
+
+namespace NARFO_BE.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ActivitiesController : ControllerBase
+    {
+        private readonly narfoContext _context;
+
+        public ActivitiesController(narfoContext context)
+        {
+            _context = context;
+        }
+
+        // GET: api/Activities
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<Activity>>> GetActivity()
+        {
+            return await _context.Activity.ToListAsync();
+        }
+
+        // GET: api/Activities/5
+        [HttpGet("{id}")]
+        public async Task<ActionResult<Activity>> GetActivity(int id)
+        {
+            var activity = await _context.Activity.FindAsync(id);
+
+            if (activity == null)
+            {
+                return NotFound();
+            }
+
+            return activity;
+        }
+
+        // PUT: api/Activities/5
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutActivity(int id, Activity activity)
+        {
+            if (id != activity.ActiviryId)
```

Comment:
spelling error. multiple instances. Refactor when changing it

Reply:
someone fucked when designing the database..... we pulled that from the client's database tables